src/rgw/rgw_acl_s3.h

```diff
@@ -14,7 +14,7 @@
 #include "rgw_acl.h"
 
 class RGWUserCtl;
-namespace rgw { namespace sal { class RGWStore; } }
+namespace rgw { namespace sal { class Store; } }
```

Comment:
food for thought: introduce a rgw_sal_fwd.h that provides forward declarations for all of the types

Reply:
Good idea.